EMI updates (#7603)
fishtown-analytics/docs.getdbt.com: website/docs/docs/explore/external-metadata-ingestion.md

## Configuration instructions

> **Opt into feature and enable external metadata ingestion:**

Reviewer: ```suggestion
### Enable external metadata ingestion
```

1. Click your account name at the bottom of the left-side menu and click **[Account settings](/docs/cloud/account-settings)**.
2. Under Account information, go to **Settings** and click **Edit** at the top right corner of the page.
3. Select the **Ingest external metadata in dbt Catalog (formerly dbt Explorer)** option (if not already enabled).

> **Locate/create the warehouse connection**

Reviewer: I suggest we use this section title because the steps below do not stop at locating or creating the connection:
```suggestion
### Configure the warehouse connection
```

> 4. Go back to **Account settings**.

Reviewer: ```suggestion
1. Go to **Account settings**.
```

> 5. Click **Connections** from the left hand side panel and select an existing connection or create a **New connection**.

Reviewer: ```suggestion
2. Click **Connections** from the left-hand side panel.
3. Select an existing connection or create a [**New connection**](/docs/cloud/connect-data-platform/connect-snowflake) where you want to ingest metadata from.
```

Reviewer: There will be additional steps for the user if they create a new connection. Can we add those here? Maybe just a short note that if the user will create a new connection, they would have to follow the steps to create a new connection (add a relevant link).

Author: Hiya @luna-bianca 

I've added a link to the snowflake connection setup as Snowflake is the only adapter supported for EMI currently.

Kind Regards
Natalie

> 6. Locate or create the warehouse connection you want to ingest metadata from.

Reviewer: ```suggestion
```

> 7. Scroll to the bottom of the page and click **Add credentials** in **Platform metadata credentials**.

Reviewer: ```suggestion
4. Scroll to the bottom of the page and click **Add credentials** in **Platform metadata credentials**.
```

>     - These credentials should have warehouse-level visibility across relevant databases and schemas.

Reviewer: Step 4 does not explicitly ask users to fill out the fields -- can we add something like:
```suggestion
    - Enter the necessary credentials. These should have warehouse-level visibility across relevant databases and schemas.
```

> 8. Select the **External metadata ingestion** option.

Reviewer: ```suggestion
5. Select the **External metadata ingestion** option.
```

    - This allows metadata from this connection to populate the <Constant name="explorer" />.
    - *Optional*: Enable additional features such as **cost optimization** in the **Features** section under **Platform metadata credentials**.

> 9. Apply filters to restrict which metadata is ingested:

Reviewer: ```suggestion
6. Under **Catalog filters**, apply filters to restrict which metadata is ingested:
```

To enable external metadata ingestion:

> 1. Navigate to **[Account settings](/docs/cloud/account-settings)**.

Reviewer: ```suggestion
1. Click your account name at the bottom of the left-side menu and click **[Account settings](/docs/cloud/account-settings)**.
```

> 2. Scroll to **Settings** and select **Edit** at the top right of the page.

Reviewer: ```suggestion
2. Under **Account**, go to **Settings** and click **Edit** at the top right corner of the page.
```

> 2. Under **Account**, go to **Settings** and click **Edit** at the top right corner of the page.

Author: ```suggestion
2. Under Account information, go to **Settings** and click **Edit** at the top right corner of the page.
```

> 3. Check the box to enable **Ingest external metadata in dbt Catalog (formerly dbt Explorer)** (if not already enabled).

Reviewer: ```suggestion
3. Select the **Ingest external metadata in dbt Catalog (formerly dbt Explorer)** option (if not already enabled).
```
Use "select" for check boxes: https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md#checkboxes

> 4. Locate or create the warehouse connection you want to ingest metadata from.

Reviewer: I suggest we add information here how to locate/create the warehouse connection. Is this covered by Steps 5 and 6 (going to Connections and selecting existing or new connection)? If it is covered by Steps 5 and 6, then they should be substeps of Step 4 instead.

Reviewer 2: Agreed on the above! I think you mixed up some steps here: 

The order should be: 

Step 5
Step 6
Step 4
Step 7
Step 8

 Anther idea here, Technically the first three steps will be ephemeral and it's just about opting into the preview feature.  We could have a section that say "Opt into feature".  Once this becomes GA, the opt in will no longer be an issue.

Then the actual configuration instructions which are separate and those will always be there.

5. Go back to **Account settings**.
6. Click **Connections** from the left hand side panel and select existing connection or **New connection**.

> 7. Scroll to the bottom of the page and **Add credentials** in **Platform metadata credentials**.

Reviewer: ```suggestion
7. Scroll to the bottom of the page and click **Add credentials** in **Platform metadata credentials**.
```

>     - These should have warehouse-level visibility across relevant databases and schemas

Reviewer: ```suggestion
    - These credentials should have warehouse-level visibility across relevant databases and schemas.
```

> 8. Enable the option for “external metadata ingestion”.

Reviewer: ```suggestion
8. Select the **External metadata ingestion** option.
```

    - This allows metadata from this connection to populate the <Constant name="explorer" />.

>     - *Optional*: Enable additional features such as **cost optimization**.

Reviewer: Where and how to enable this feature?

9. Apply filters to restrict which metadata is ingested:
    - You can filter by **database**, **schema**, **table**, or **view**.

>     - Strongly recommend you filter by certain schemas. See [Important considerations](/docs/explore/external-metadata-ingestion#important-considerations) for more information.

Reviewer: ```suggestion
    - It is strongly recommend to filter by certain schemas. See [Important considerations](/docs/explore/external-metadata-ingestion#important-considerations) for more information.
```

    - These fields accept CSV-formatted regular expressions:
        - Example: `DIM` matches `DIM_ORDERS` and `DIMENSION_TABLE` (basic "contains" match).

>         - The wildcards: `DIM*` matches `DIM_ORDERS`, `DIM_PRODUCTS`, are supported.

Reviewer: Suggest to revise this to:
```suggestion
        - Wildcards are supported. For example: `DIM*` matches `DIM_ORDERS` and `DIM_PRODUCTS`.
```
